tests/kafkatest/services/verifiable_producer.py

```diff
@@ -55,16 +79,32 @@ def _worker(self, idx, node):
                     elif data["name"] == "producer_send_success":
                         self.acked_values.append(int(data["value"]))
 
-    @property
-    def start_cmd(self):
-        cmd = "/opt/kafka/bin/kafka-verifiable-producer.sh" \
+    def _kafka_dir(self, node):
+        if node.version == KafkaVersion.TRUNK:
```

Comment:
My only hesitation against calling it `kafka-trunk` there was because `/opt/kafka` really has whatever branch we happen to have last compiled.  `KafkaVersion.TRUNK` is misleading in that regard, too